src/Microsoft.Dnx.Compilation.CSharp/RoslynProjectReference.cs

```diff
@@ -151,7 +205,69 @@ public void EmitReferenceAssembly(Stream stream)
 
         public DiagnosticResult EmitAssembly(string outputPath)
```

Comment:
This method got super long. Move the resource generation part into another method.